src/spec/page-objects/settings-page.js

```diff
@@ -57,5 +58,13 @@ class SettingsPage {
       return val[0].click();
     });
   }
+  getDisabledDomain(){
+    return this._domainsDisabled.getText();
+  }
+  clickDisabledDomain(){
+    return this._domainsDisabled.then(function(val){

```

Comment:
No no, but, I think that the method name should reflect what it is happening, because, in the future someone could add a new test scenario with more elements, and in this case, how it will work?

Does `clickFiirstDisabledDomainButton` be more precise? or I am missing something?